packages/ui5-application-writer/test/data.test.ts

```diff
@@ -194,7 +194,7 @@ describe('Setting defaults', () => {
                 version: '1.199.0',
                 localVersion: '1.199.0',
                 minUI5Version: '1.199.0',
-                descriptorVersion: '1.60.0',
+                descriptorVersion: '1.65.0',
```

Comment:
@schreckstefan, do we know why this changed?

Reply:
@Klaus-Keller I had to adapt the test like this, after the switch to ui5/manifest 1.66. Frankly speaking, I don't know why the testee (mergeUI5) returns values like this, I did not program this. 
Maybe one of the authors can comment? @IainSAP @longieirl @tobiasqueck